filter.js

```diff
@@ -1,9 +1,35 @@
-let arr = [5, 3, 8, 1];
+// Filter homework new array
+const arr = [5, 3, 8, 1];
 function filterRange(arr, a, b){
   return arr.filter((item) => item >= a && item <= b)
 };
-let filtered = filterRange(arr, 1, 4);
+const filtered = filterRange(arr, 1, 4);
 
-alert( filtered ); // 3,1 (совпадающие значения)
+console.log( filtered ); // 3,1 (совпадающие значения)
 
-alert( arr );
\ No newline at end of file
+console.log( arr );
+
+// Filter homework
+let newArr = [5, 3, 8, 1]; 
+function filterRangeInPlace(newArr, a, b) { 
+  return newArr.filter((item) => item >= a && item <= b); 
+} 
+newArr = filterRangeInPlace(newArr, 1, 4); // удалены числа вне диапазона 1..4 
```

Comment:
this function doesn't filter in a place, it returns a new array. You should filter the existing array